src/ClientContextInterface.php

```diff
@@ -45,6 +45,7 @@
 use Predis\Command\Container\Search\FTCONFIG;
 use Predis\Command\Container\Search\FTCURSOR;
 use Predis\Command\Container\XGROUP;
+use Predis\Command\Redis\VADD;
```

Comment:
Is this needed?

Reply:
Yes, we refer to a constant associated with this class as a default argument.

`* @method $this vadd(string $key, string|array $vector, string $elem, int $dim = null, bool $cas = false, string $quant = VADD::QUANT_DEFAULT, ?int $BEF = null, string|array $attributes = null, int $numlinks = null)`